### PR TITLE
Update README.md with new text and new logo

### PR DESCRIPTION
**Pull request by anedos-dfinity**
Oisy's README.md was outdated and with the new logo, I thought I would take the opportunity and improve our messaging.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<div align="center" style="display:flex;flex-direction:column;">
+<div style="display:flex;flex-direction:column;">
   <a href="https://oisy.com/">
-    <img src="./src/frontend/static/images/meta-share.jpg" alt="Oisy Wallet logo" role="presentation"/>
+    <img src="./oisy-github-image.jpg" alt="Oisy Wallet logo" role="presentation"/>
```
Comment (peterpeterparker) [inline README.md:3]: Please remove this image and use the upcoming new meta-share provided in #1906 (once fixed and merged)

File: README.md [hunk 2]
```diff
@@ -14,31 +14,26 @@
 
 ---
 
-## What is the Oisy Wallet
+## What is the Oisy wallet
```
Comment (peterpeterparker) [inline README.md:17]: No more "Wallet" with a capital letter?
Should we update the code base to reflect this change?

**Review by peterpeterparker — COMMENTED** [review]
Two comments.

Note that I did not read the provided text changes.

**Comment by peterpeterparker** [comment]
Also please fix PR title by providing a context and run the formatter.

File: README.md (hunk 2 above)
Comment (anedos-dfinity) [reply README.md:17]: Let's do it gradually, but yes, I personally have a slight preference for `Oisy wallet` over `Oisy Wallet`

**Pull request closed without merging**